server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl

```diff
@@ -15,6 +15,12 @@ CREATE TABLE "ORGANIZATIONS" (
 CREATE UNIQUE INDEX "PK_ORGANIZATIONS" ON "ORGANIZATIONS" ("UUID");
 CREATE UNIQUE INDEX "ORGANIZATION_KEY" ON "ORGANIZATIONS" ("KEE");
 
+CREATE TABLE "ORGANIZATION_MEMBERS" (
+  "UUID" VARCHAR(40) NOT NULL PRIMARY KEY,
```

Comment:
composed primary keys are base element of SQL.
Even though you don't want to use composite primary keys, the uuid column is useless and should be dropped. The unique index is enough.